code/game/objects/structures/window.dm

```diff
@@ -207,6 +207,7 @@
 				to_chat(user, "<span class='notice'>You begin to [anchored ? "unscrew the window from":"screw the window to"] the floor...</span>")
 				if(do_after(user, decon_speed*I.toolspeed, target = src, extra_checks = CALLBACK(src, .proc/check_anchored, anchored)))
 					anchored = !anchored
+					air_update_turf(TRUE)
```

Comment:
I can't figure out why you only update air on the non-reinforced window after screwdrivering